composer.json

```diff
@@ -24,9 +24,9 @@
   "license": "MIT",
   "require": {
     "php": ">=7.2",
-    "typo3/cms-core": "^10.0",
-    "friendsoftypo3/headless": "^2.0",
-    "gridelementsteam/gridelements": "^10.0"
+    "typo3/cms-core": "^11.5",
+    "friendsoftypo3/headless": "^3.0",
+    "gridelementsteam/gridelements": "dev-11.0.0-dev"
```

Comment:
@kitzberger I believe it is sufficient to only require `11.0.0-dev` literally because Jo declares the version in the composer file explicitly to that version (see: https://gitlab.com/coderscare/gridelements/-/blob/master/composer.json#L5).

Furthermore I believe that `dev-11.0.0-dev` might not work because a branch `11.0.0-dev` does not exist. Can you confirm?

Reply:
@schloram, I don't think that the version property with the composer.json of EXT:gridelements can be addressed at all. Only branch names and tags can. At least that's how I understand it.

Therefore I'm still voting for the wildcard because all EXT:headless_gridelements needs is a TYPO3 11.5 compatible version of EXT:gridelements ;-)

Reply:
@kitzberger The `*` wildcard contraint is a no-go for me. The only case it might be conceivable is a private project with local composer packages in the same repo.
for public repos it's just too fragile. I never know what a required package drops in future versions so this might result in errors which must always be "hotfixed". With this it might look like more "work" to always update the gridelements contraint but we are on the safe side regarding breaking changes.

The only thing we could think about is that for our `3.0.0-dev` (master branch) version we could require `gridelementsteam/gridelements:dev-master`. But as soon as we release a proper version we have to set the contraint to a specific version tag.

Reply:
@schloram, that's ok for me, you're the boss here ;-) Maybe squash the commits when merging? ;-)